pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatorTest.java

```diff
@@ -803,6 +804,32 @@ public void verifyChecksumAfterReplication() throws Exception {
         reader2.closeAsync().get();
     }
 
+    @Test
+    public void testReplicatorWithPartitionedTopic() throws Exception {
+        final String namespace = "pulsar/partitionedNs-" + UUID.randomUUID();
+        final String persistentTopicName = "persistent://" + namespace + "/partTopic" + UUID.randomUUID();
+
+        admin1.namespaces().createNamespace(namespace);
+        admin1.namespaces().setNamespaceReplicationClusters(namespace, Sets.newHashSet("r1", "r2", "r3"));
+        // Create partitioned-topic from R1
+        admin1.topics().createPartitionedTopic(persistentTopicName, 3);
+        // List partitioned topics from R2
+        Awaitility.await().untilAsserted(() -> assertNotNull(admin2.topics().getPartitionedTopicList(namespace)));
+        Awaitility.await().untilAsserted(() -> assertEquals(
+                admin2.topics().getPartitionedTopicList(namespace).get(0), persistentTopicName));
+        assertEquals(admin1.topics().getList(namespace).size(), 3);
```

Comment:
@315157973 Could you please also add a check for checking we have 3 partitions in the cluster-2(using admin-1)

```
assertEquals(admin2.topics().getList(namespace).size(), 3);
```

Reply:
Ok i will add this test

Reply:
Hello @codelipenghui 
`admin.topics().getList` reads the data under the /managed-ledgers node, which belongs to localZK, so the remote cluster cannot perceive it. This PR mainly fixes the remote replication problem of updatePartitions in issue#10673.

Now the CreatePartitionTopic interface does not have geo-related logic. If we want to add it, we need to add parameters to avoid circular replication.  I will open another PR to support  replication in CreatePartitionTopic.